modules/ROOT/pages/clauses/create.adoc

```diff
@@ -196,3 +205,17 @@ Nodes created: 2 +
 Properties set: 4
 |===
 
+[[insert-as-synonym-of-create]]
+== `INSERT` as a synonym of `CREATE`
+
+`INSERT` can be used as a synonym to `CREATE` for creating nodes and relationships.
+`INSERT` requires that multiple labels are separated by an ampersand `&` and not by colon `:`.
+
+.Query
+[source, cypher, indent=0]
+----
+INSERT (tom:Person&Actor&Director {name: 'Tom Hanks'})
```

Comment:
@recrwplay i noticed that in the code block for this line in the output HTML only "Person" is highlighted blue, but not "Actor" nor "Director". how can we fix that?

![Screenshot from 2024-01-26 16-13-31](https://github.com/neo4j/docs-cypher/assets/156673635/faa827f7-89b4-4b17-9125-76f716559b40)

Reply:
We use `highlightjs` with `highlightjs-cypher` for the highlighting, but if this new syntax it won't be aware of it yet.